composer.json

```diff
@@ -3,6 +3,7 @@
 		"php": ">=5.2.7"
 	},
 	"require-dev": {
-		"satooshi/php-coveralls": "dev-master"
+		"satooshi/php-coveralls": "^2.0",
+		"squizlabs/php_codesniffer": "^3.2"
```

Comment:
Why is this being added, the `dev-lib` will do all WPCS sniffing and has a pre-commit hook, as well.

Reply:
Good do know, thanks :) 

I have 0 knowledge of how this works so figuring it out as we go.

Reply:
If you want to have a knowledge transfer meeting to discuss how it all works let me know.

Reply:
Knowledge share would be rad. I'll hit you up